app/assets/javascripts/discourse/app/lib/sidebar/admin-nav-map.js

```diff
@@ -40,9 +40,9 @@ export const ADMIN_NAV_MAP = [
     links: [
       {
         name: "admin_about_your_site",
-        route: "adminSiteSettingsCategory",
-        routeModels: ["required"],
-        query: { filter: "" },
+        route: "adminConfig.about",
+        // routeModels: ["required"],
+        // query: { filter: "" },
```

Comment:
Is the admin nav sidebar still experimental? Should I keep this as-is until the about config area is more built out?

Reply:
The nav sidebar should no longer be considered experimental, we've enabled it on almost all sites now. Definitely keep this as is until the new about page is complete :+1: